filebeat/module/shorewall/log/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- name: network
+  type: group
+  description: >
+    Shorewal log files
+  fields:
+    - name: in
+      type: keywork
```

Comment:
Description of fields is "mandatory" and "extended" even if the potential target user is a Shorewall expert